test/functional/executor.py

```diff
@@ -56,6 +58,10 @@ def run(self):
         passed = 0
         for test in self.tests:
             cmd = [self.executable, *test.get_command()]
+            if self.verbose:
+                print("TEST: ", test.get_name())
+                print("DESCRIPTION: ", test.get_description())
+                print("RUNNING: ", *cmd)
             subprocess.run(cmd, stdout=subprocess.DEVNULL,
                            stderr=subprocess.DEVNULL)
```

Comment:
Well, in this case verbose means the 'test verbose' which results in printing the test name, brief description and the command which is issued. I like this format because if needed you can simply copy/paste the command in the terminal and quickly modify it for your own purposes.

The application verbose mode (-v passed to the tested application) is configured by the XML tags.